source/common/common/perf_annotation.cc

```diff
@@ -108,7 +109,7 @@ std::string PerfAnnotationContext::toString() {
             : std::to_string(
                   std::chrono::duration_cast<std::chrono::nanoseconds>(stats.total_).count() /
                   count));
-    columns[3].push_back(absl::StrCat("", stats.stddev_.computeStandardDeviation()));
+    columns[3].push_back(absl::StrFormat("%g", stats.stddev_.computeStandardDeviation()));
```

Comment:
I think most devs wont' know this and we are going to wind up with issues in the future. WDYT about some type of `doubleToString` helper?

Reply:
Makes sense, I played around with `std::to_string` for doubles to see if we can just leverage that but the format is a bit different - always six digits after the decimal including trailing zeros (`%f`) while `%g` trims trailing zeros.